File: resourcespoet/src/test/java/com/commit451/resourcespoet/TestUtil.java
package com.commit451.resourcespoet;

import org.junit.Assert;

/**
 * Helps with tests
 */
public class TestUtil {

    public static void assertEquals(String xmlFileName, ResourcesPoet poet) throws Exception {
        poet.indent(true);
        String text = Util.getFileText(xmlFileName);

        String writtenContent = poet.build();
        System.out.println(writtenContent);

        Assert.assertEquals(Util.trimtrimtrim(text), Util.trimtrimtrim(writtenContent));
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/TestUtil.kt
package com.commit451.resourcespoet

import org.junit.Assert

/**
 * Helps with tests
 */
object TestUtil {

    fun assertEquals(xmlFileName: String, poet: ResourcesPoet) {
        poet.indent(true)
        val text = Util.getFileText(xmlFileName)

        val writtenContent = poet.build()
        println(writtenContent)

        Assert.assertEquals(Util.trimtrimtrim(text), Util.trimtrimtrim(writtenContent))
    }
}
